### DISCUSSION TITLE
Perms renovation

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#17] [Ideas]
So this is basically a suggestion to remove entirely admins perms to dupe or copy your props or acces your e2 withour permission, I know admins are trusted people but this problem has happend in the past and its really not cool to get your e2 used by someone without permission.

**Comment by Kilometres** [comment]
Admins can't dupe or copy your props so I think you might be a bit confused there. 

It also notifies you in chat if someone opens your E2 and there is no way for an admin to suppress this.

You said this has happened in the past, who was doing this in the past?

**Reply by ELPatitoLoco** [reply]
Admins can in fact copy or dupe because the other day a admin that I'm not gonna mention just duped the shit out of my e2 dupe (I have a couple props and a e2 saved in a dupe)and placed it, and I think you can change the perms for e2 acces if not maybe I could just fork it myself but maybe to much of a mess.

**Reply by Kilometres** [reply]
Copying E2s with AdvDupe2 is because admins have prop protection perms (which are needed to prevent crasher/laggers) so unfortunately I don't think there will be a solution to that. And if you can't stop it you may as well keep accessing it with toolgun (with notification) as it makes it **a lot** easier when helping people with e2.

The unfortunate truth is whenever playing on a server, your e2s are not safe, A server has full perms to steal your e2s then wipe ur data folder clean. If this is a large concern for you, play on trusted servers, backup your e2s, and attempt to obfuscate.

**Reply by Derpius** [reply]
As Kilo said, this is based on prop protection, and admins _currently_ need prop protection perms to mitigate lag and crashes

_However_, if we removed these permissions (which would require forking nadmod to remove isadmin checks most likely) we could instead provide a suite of ulx commands as we already do

Alternatively we could still fork nadmod, but limit admin bypass to only removing props

**Reply by Kilometres** [reply]
Maybe this could be kept in mind for playerbase.

**Reply by Derpius** [reply]
Not sure patito wants to wait until 2040

In all seriousness though, should be an easy change to do it in nadmod for now. I’ll add it to the possible contributions list

**Comment by Derpius** [comment]
A solution to resolve this has been added to the beginner section of https://github.com/orgs/TAServers/discussions/16
